index.bs

```diff
@@ -345,7 +350,7 @@ class AV1LayeredImageIndexingProperty extends ItemProperty('a1lx') {
 
 <h6 id="layered-image-indexing-property-semantics" class="no-toc">Semantics</h6>
 
-<dfn noexport>layer_size</dfn> indicates the number of bytes corresponding to each layer in the item payload, except for the last layer. Values are provided in increasing order of <code>[=spatial_id=]</code>. A value of zero means that all the layers except the last one have been documented and <assert>following values shall be 0</assert>. <assert>The number of non-zero values shall match the number of layers in the image minus one.</assert>.
+<dfn noexport>layer_size</dfn> indicates the number of bytes corresponding to each layer in the item payload, except for the last layer. Values are provided in increasing order of <code>[=spatial_id=]</code>. A value of zero means that all the layers except the last one have been documented and <assert>following values shall be 0</assert>. <assert>The number of non-zero values shall match the number of layers in the image minus one.</assert>
```

Comment:
Add back the period (.) at the end of this paragraph.

Reply:
Ah, I didn't notice that the last sentence of this paragraph already ends with a period, so you deleted an extraneous period (after `</assert>`).